test/flat-packer-base/__snapshots__/flat-packer.test.js.snap

```diff
@@ -195,6 +141,123 @@ function __fastpack_import__(f) {
   });
 }
 
+/* index$$_$$dynamic */
+
+__fastpack_import__($w__dep$$_$$esm$$_$$named).then(named => {
+  console.log(named);
+});
+
+const $n__index$$_$$dynamic = { exports: {} };
+
+function $w__dep$$_$$esm$$_$$named() {
+
+/* dep$$_$$esm$$_$$named */
+
+const $e__dep$$_$$esm$$_$$named__a = 1;
+const $e__dep$$_$$esm$$_$$named__b = 2;
+
+const $n__dep$$_$$esm$$_$$named = { exports: {a: $e__dep$$_$$esm$$_$$named__a, b: $e__dep$$_$$esm$$_$$named__b} };
+
+return $n__dep$$_$$esm$$_$$named.exports;
+}
+})()
+"
+`;
+
+exports[`flat packer with dynamic esm dep: target: cjs 1`] = `
+"var __DEV__ = false;
+var __fastpack_cache__ = {};
+
+function __fastpack_require__(f) {
+  if (__fastpack_cache__[f.name] === undefined) {
+    __fastpack_cache__[f.name] = f();
+  }
+  return __fastpack_cache__[f.name];
+}
+
+function __fastpack_import__(f) {
+  return new Promise((resolve, reject) => {
+    try {
+      resolve(__fastpack_require__(f));
+    } catch (e) {
+      reject(e);
+    }
+  });
+}
+
+/* index$$_$$dynamic */
+
+__fastpack_import__($w__dep$$_$$esm$$_$$named).then(named => {
+  console.log(named);
+});
+
+const $n__index$$_$$dynamic = { exports: {} };
+module.exports = $n__index$$_$$dynamic.exports;
+
+function $w__dep$$_$$esm$$_$$named() {
+
+/* dep$$_$$esm$$_$$named */
+
+const $e__dep$$_$$esm$$_$$named__a = 1;
+const $e__dep$$_$$esm$$_$$named__b = 2;
+
+const $n__dep$$_$$esm$$_$$named = { exports: {a: $e__dep$$_$$esm$$_$$named__a, b: $e__dep$$_$$esm$$_$$named__b} };
+module.exports = $n__dep$$_$$esm$$_$$named.exports;
+
+return $n__dep$$_$$esm$$_$$named.exports;
+}
+"
+`;
+
+exports[`flat packer with dynamic esm dep: target: esm 1`] = `
+"var __DEV__ = false;
+var __fastpack_cache__ = {};
+
+function __fastpack_require__(f) {
```

Comment:
`__fastpack_require__` seems to be redundant. We have treat all `require()` calls as static dependencies at the moment. Maybe we could rip it off entirely?